angular/README.md

```diff
@@ -1,15 +1,64 @@
-# Angular and Wootric Tutorial
+# `angular` — the wootric configured project for Angular apps
 
-![](/angular/angular-wootric-screenshot.png)
+This project is an application skeleton for a typical [Wootric](https://www.wootric.com/) setup web app. You can use it
+to quickly bootstrap your wootric setup angular webapp projects.
 
-## Requirements
+The app doesn't do much, just shows how to setup the wootric snippet for Anuglar projects.
 
-- [Angular CLI](https://cli.angular.io)
+## Getting Started
 
-## Instructions
+To get you started simply follow the instructions below:
 
-- `git clone git@github.com:Wootric/spa-examples.git`
-- `cd spa-examples/angular`
-- Open `src/app/wootric-survey.directive.ts`.
-- Replace the account token on line 14 with your account token.
-- Run the command `ng serve` to start the development server and see your survey.
+### Prerequisites
+
+You need git to clone the [spa-examples](https://github.com/Wootric/spa-examples) repository. You can get git from [here](https://git-scm.com/).
+
+We also use a number of Node.js tools to install `angular-cli`. You must have Node.js and its package manager (npm) installed. You can get them from [here](https://nodejs.org/). Then install Angular CLI globally with command `npm install -g @angular/cli
+
+### Clone `spa-examples`
+
+Clone the `spa-examples` repository using git:
+
+```
+git clone https://github.com/Wootric/spa-examples.git
+cd spa-exaples
+cd angular
+npm install
+````
+
+### Setup Wootric (using angular directive)
+1. Create `angular directive` with command `ng generate directive wootric`
```

Comment:
<img width="707" alt="Screen Shot 2019-06-28 at 12 33 24 PM" src="https://user-images.githubusercontent.com/10385848/60366637-f2de9000-99a0-11e9-808e-cb292600ead2.png">

When I run this command I get an error that the file already exists. I think you should change the setup instruction to edit this file instead of trying to generate it.

Reply:
Umm makes sense ( I actually thought of the process if someones want to start with the beginning of creating directive but it miss-matched here). I now have removed this step altogether to clear the confusion.